pythoncode/FortiusAntBody.py

```diff
@@ -318,9 +339,14 @@ def Runoff(self):
         logfile.Console('Runoff not implemented for Simulated trainer or Tacx i-Vortex')
         return False
 
-    TacxTrainer.SetPower(100)
+    #---------------------------------------------------------------------------
+    # Initialize
+    #---------------------------------------------------------------------------
+    TacxTrainer.SetPower(clv.RunoffPower)
     rolldown        = False
     rolldown_time   = 0
+    ShortMessage    = TacxTrainer.Message + " | Runoff - "
+    ShortMessage    = "Tacx Trainer Runoff - "
```

Comment:
Which one of the messages is meant to be displayed? Currently the latter will take precedence.

Reply:
Agree; first message uses the "standard" but is quite long.
Second message is as suggested by CyclingFlow and indeed replaces the first (could be commented for clarity)